includes/admin/payments/orders.php

```diff
@@ -1026,39 +1021,13 @@ function edd_order_details_attributes( $order ) {
 								<?php esc_html_e( 'Hour', 'easy-digital-downloads' ); ?>
 							</label>
 
-							<?php
-							echo EDD()->html->select(
-								array(
-									'name'             => 'edd-payment-time-hour',
-									'id'               => 'edd-payment-time-hour',
-									'options'          => edd_get_hour_values(),
-									'selected'         => $order_date->format( 'H' ),
-									'chosen'           => true,
-									'class'            => 'edd-time edd-form-group__input',
-									'show_option_none' => false,
-									'show_option_all'  => false,
-								)
-							); // WPCS: XSS ok.
-							?>
+							<input type="number" min="0" max="24" step="1" name="edd-payment-time-hour" id="edd-payment-time-hour" value="<?php echo esc_attr( $order_date->format( 'H' ) ); ?>" />
```

Comment:
The related labels for the hour and minute inputs need to be updated to match the new ID (`EDD()->html->select()` converts `-` to `_` in the ID, which is no longer applicable here).